uniflow/flow/config.py

```diff
@@ -362,6 +361,51 @@ class RaterForClassificationBedrockClaudeConfig(RaterConfig):
     )
 
 
+@dataclass
+class RaterForClassificationHuggingfaceConfig(RaterConfig):
+    """
+    The configuration primarily focuses on setting up the parameters for utilizing Huggingface model to
+    evaluate the correctness of answers in relation to given questions and contexts.
+
+    Attributes:
+        flow_name (str): Name of the rating flow, default is "RaterFlow".
+        model_config (ModelConfig): Configuration for the huggingeface model. Includes model_name("mistralai/Mistral-7B-Instruct-v0"),
+                                    model_server ("HuggingfaceModelServer"), batch_size (1), neuron (False),
+                                    load_in_4bit (False), load_in_8bit (True)
+        label2score (Dict[str, float]): Mapping of labels to scores, default is {"Yes": 1.0, "No": 0.0}.
+        guided_prompt_template (GuidedPrompt): Template for guided prompts used in rating. Includes instructions
+                                               for rating, along with examples that detail the context, question,
+                                               answer, label, and explanation for each case.
+    """
+
+    model_config: ModelConfig = field(default_factory=HuggingfaceModelConfig())
```

Comment:
HuggingFace model is refactored in this https://github.com/CambioML/uniflow/pull/110.
- it introduce 
[text](https://github.com/CambioML/uniflow/blob/main/uniflow/flow/config.py#L122-L124) response format and [json](https://github.com/CambioML/uniflow/blob/main/uniflow/flow/config.py#L158-L160) response format to better control huggingface model especially mistral model.
- it also introduce a new argument `response_start_key` for your case, i believe model generated response starting with `explanation` which should be the start key. 

Please up you your config.py properly to address change in https://github.com/CambioML/uniflow/pull/110 and then in your notebook add use case for both text and json mode.

Reply:
@Panzy-18 Any progress on addressing this comment?